dspace-api/src/main/java/org/dspace/core/NewsServiceImpl.java

```diff
@@ -35,8 +37,23 @@
 	private List<String> acceptableFilenames;
 	
 	public void setAcceptableFilenames(List<String> acceptableFilenames) {
-		this.acceptableFilenames = acceptableFilenames;
-	}
+        this.acceptableFilenames = addLocalesToAcceptableFilenames(acceptableFilenames);
+    }
+
+    private List<String> addLocalesToAcceptableFilenames(List<String> acceptableFilenames){
+        ConfigurationService configurationService =  DSpaceServicesFactory.getInstance().getConfigurationService();
```

Comment:
The NewsServiceImpl is a bean you can autowire the configuration service in (you don't have to use the factory to retrieve an instance.

An example of autowiring: https://github.com/DSpace/DSpace/blob/master/dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java#L55